src/components/TodoApp.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { useLocalStorage } from '../utils/useLocalStorage';
+import { Footer } from './Footer';
+import { Header } from './Header';
+import { TodoList } from './TodoList';
+// import { ErrorNotification } from './ErrorNotification';
+import { Error } from '../types/Error';
+
+export const TodoApp: React.FC = () => {
+  const [todos, setTodos] = useLocalStorage('todos', []);
+  // const [errorMessage, setErrorMessage] = useState<string>('');
+  const [hidden, setHidden] = useState(true);
+  const [error, setError] = useState('');
+  const closeErrorMessage = () => {
+    setHidden(false);
+  };
```

Comment:
You could just clear error `setError('')` and after check if error is empty or not to show or hide the error message